spec/pardot/authentication_spec.rb

```diff
@@ -32,6 +32,45 @@ def authenticate
       authenticate
       @client.authenticated?.should == true
     end
+
+    it "should use version 3" do
+      authenticate
+      @client.version.to_i.should == 3
+    end
+    
+  end
+
+  describe "authenticateV4" do
```

Comment:
The BDD equivalent would be `describe "authenticate" ... context "version 4"`  Same thing for the client / http `get` and `post` specs below.  Up to you if you want to update now, later, or never...as long as it's tested, it's fine with the current spec.  We can always refactor later once it gets hairy.